tests/common/ixia/ixia_helpers.py

```diff
@@ -0,0 +1,439 @@
+    traffic_statistics = session.StatViewAssistant(stat_view_name)
+    ixnetwork.info('{}\n'.format(traffic_statistics))
+    return traffic_statistics
+
+
+def stop_traffic(session):
+    """ This function stops all the IxNetwork traffic items configured 
+        on all the ports.
+    Args:
+        session (obj): IxNetwork session object.
+
+    Returns:
+        None.   
+    """
+    ixnetwork = session.Ixnetwork
+    ixnetwork.Traffic.StopStatelessTrafficBlocking()
+
+
+def start_traffic(session):
+    """ This function starts all the IxNetwork traffic items configured 
+        on all the ports.
+    Args:
+        session (obj): IxNetwork session object.
+
+    Returns:
+        None.   
+    """
+    ixnetwork = session.Ixnetwork
+    """ Apply traffic to hardware """
+    ixnetwork.Traffic.Apply()
+    """ Run traffic """
+    ixnetwork.Traffic.StartStatelessTrafficBlocking()
+
+
+def create_ip_traffic_item_using_wizard_arguments (
+    session,
+    src_start_port,
+    src_port_count,
+    src_first_route_index,
+    src_route_count,
+    dst_start_port,
+    dst_port_count,
+    dst_first_route_index,
+    dst_route_count,
+    name='example_traffic',
+    traffic_type='ipv4') :
+
+    """
+    This function creates a traffic item where source and destination ports 
+    belong to same IxNetwork topology-object. Since source and destination 
+    belong to same topology, source and destination endpoints may be
+    selected by selecting starting source port, source port count, first
+    route address index on the source port, source route count, destination
+    start port, destination port count, destination first-route index,
+    and destination route count.
+
+    Args:
+        session (obj): IxNetwork session object. 
+        src_start_port (int): The start port number.
+        src_port_count (int): The number of ports involved in sending traffic 
+            starting from src_start_port number. Example, if the start port is 
+            port2 and port2 to port5 is sending traffic then src_start_port = 2
+            and src_port_count = 3.   
+        src_first_route_index (int): The first route address index.
+        src_route_count (int): Number of routes starting from the 
+            src_first_route_index.
+        dst_start_port (int): The first destination port number.  
+        dst_port_count (int): Number of ports involved in receiving the traffic
+            starting from dst_start_port number. Example, if rhe rx port is
+            port6 and port7 then dst_start_port = 6 and dst_port_count = 2
+        dst_first_route_index (int): The first destination IP index.
+        dst_route_count (int): Number of destination IPs starting from
+           dst_first_route_index.
+        name (str, optional): Name of the traffic item. Default name is
+           'example_traffic'.
+        traffic_type (str, optional): Type of the ip source and destination
+        (ipv4/ipv6). Default traffic_type is 'ipv4'.
+
```

Comment:
The meanings of 'src_first_route_index', 'src_route_count', 'dst_fist_route_index' and 'dst_route_count' are still unclear to me.

Reply:
I have updated these comments